### PR TITLE
MINOR: Refactor tag key for store level metrics

### PR DESCRIPTION
**Pull request by cadonna**
The tag key for store level metrics specified in StreamsMetricsImpl
is unified with the tag keys on thread and task level.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by cadonna** [comment]
Call for review: @guozhangwang @vvcephei @mjsax @ableegoldman @bbejeck @abbccdda

**Review by ableegoldman — APPROVED** [review]
LGTM

File: streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java [hunk 1]
```diff
@@ -292,6 +292,9 @@ public void shouldAddMetricsOnAllLevels() throws Exception {
             .to(STREAM_OUTPUT_4);
         startApplication();
 
+        for (final Metric metric : new ArrayList<Metric>(kafkaStreams.metrics().values())) {
+            System.out.println(metric.metricName());
```
Comment (bbejeck) [inline streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java]: Is this intentional?

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredWindowStore.java [hunk 2]
```diff
@@ -75,9 +76,9 @@ public void init(final ProcessorContext context,
         metrics = (StreamsMetricsImpl) context.metrics();
 
         taskName = context.taskId().toString();
-        final String metricsGroup = "stream-" + metricScope + "-metrics";
-        final Map<String, String> taskTags = metrics.tagMap("task-id", taskName, metricScope + "-id", "all");
-        final Map<String, String> storeTags = metrics.tagMap("task-id", taskName, metricScope + "-id", name());
+        final String metricsGroup = "stream-" + metricsScope + "-state-metrics";
```
Comment (bbejeck) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredWindowStore.java]: Seems like we have moved away from `String` literals and using constants.  Any reason this is different here?

**Review by bbejeck — APPROVED** [review]
Thanks @cadonna.  Just two minor nits, otherwise LGTM

File: streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java (hunk 1 above)
Comment (cadonna) [reply streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java]: Ouch! Of course that is not intentional. Deleted!

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredWindowStore.java (hunk 2 above)
Comment (cadonna) [reply streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredWindowStore.java]: Not really! Replaced literals with constants.

**Comment by cadonna** [comment]
In JDK 11/Scala 2.12 the following test failed:
```
kafka.api.SaslSslAdminClientIntegrationTest.testElectUncleanLeadersForAllPartitions
```

Retest this, please

**Comment by bbejeck** [comment]
Merged #7257 into trunk.

**Pull request merged**